nuget/Microsoft.AspNetCore.AspNetCoreModuleV2.props.in

```diff
@@ -0,0 +1,13 @@
+<Project>
```

Comment:
Why props.in?

Reply:
that the pattern we use in other repos for GenerateFileFromTemplate inputs.